runtime/oti/j9nonbuilder.h

```diff
@@ -5658,6 +5658,9 @@ typedef struct J9CInterpreterStackFrame {
 	U_8 jitFPRs[16 * 8]; /* fpr0-15 */
 #endif /* J9VM_ENV_DATA64 */
 #elif defined(J9VM_ARCH_X86) /* J9VM_ARCH_ARM */
+#if defined(J9VM_ENV_DATA64) && defined(WIN32)
+	UDATA arguments[4]; /* outgoing arguments shadow */
+#endif /*J9VM_ENV_DATA64 && WIN32*/
 	UDATA vmStruct;
 	UDATA machineBP; /* JIT has this offset hard-coded - do not move this field */
```

Comment:
By adding the `arguments` field above have you changed the offset for this field? If so does it require JIT changes as well?

Reply:
nvm I missed the changes in #4101

Reply:
Yes, #4101 ensures JIT picks the correct offset.